### ISSUE TITLE
Error:"not an authority that can be resolved" in remote connection from url

### ISSUE THREAD

**Issue by folger-fan** [microsoft/vscode-remote-release#8700]
The "not an authority that can be resolved" error looks like https://github.com/microsoft/vscode/issues/187202. Please give the latest VS Code Insiders build a try where this should be fixed (https://code.visualstudio.com/insiders/) and open a new issue if it persists. Thanks.

_Originally posted by @chrmarti in https://github.com/microsoft/vscode-remote-release/issues/8687#issuecomment-1630237628_

**Comment by chrmarti** [comment]
> The "not an authority that can be resolved" error looks like [microsoft/vscode#187202](https://github.com/microsoft/vscode/issues/187202). Please give the latest VS Code Insiders build a try where this should be fixed (https://code.visualstudio.com/insiders/) and open a new issue if it persists. Thanks.

Thank for your reply.  I got The same error after use the latest VS Code Insiders build.  This is an example remote url of my case :
```
vscode://vscode-remote/ssh-remote+root@101.135.113.22:22/workspace
```

_Originally posted by @loviselu in https://github.com/microsoft/vscode-remote-release/issues/8687#issuecomment-1630271150_

**Comment by connor4312** [comment]
Duplicates https://github.com/microsoft/vscode/issues/187202, fixed on Inisders already
